src/main/java/com/rapid7/logback/LogentriesAppender.java

```diff
@@ -240,7 +249,13 @@ public void setLayout(Layout<ILoggingEvent> layout) {
     @Override
     protected void append(ILoggingEvent event) {
         // Render the event according to layout
-        String formattedEvent = layout.doLayout(event);
+        byte[] encodedEvent = encoder.encode(event);
+        String formattedEvent;
+        try {
+            formattedEvent = new String(encodedEvent, "UTF-8");
```

Comment:
Can we use this? I believe it removes the need for the try-catch.

```suggestion
            formattedEvent = new String(encodedEvent, StandardCharsets.UTF_8);
```

Reply:
👍